src/app.js

```diff
@@ -0,0 +1,8 @@
+// returns the nth fibonacci sequence f0=0, f1=1, f2=1, f3=3, ...
```

Comment:
Da får du tenke ut noe lurt, for nå skjønner jeg ikke hva de skal gjøre om alt allerede er gjort.